domains/gh0stp4wz.json

```
@@ -5,24 +5,13 @@
   },
   "records": {
     "CNAME": "GH0STP4WZ.github.io",
-    "TXT": [
-     "zoho-verification=zb43811690.zmverify.zoho.eu",
-     "v=spf1 include:zohomail.com ~all"
-   ],
-      "MX": [
-        {
-          "target": "mx.zoho.eu",
-          "priority": 10
-        },
-        {
-          "target": "mx2.zoho.eu",
-          "priority": 20
-        },
-        {
-          "target": "mx3.zoho.eu",
-          "priority": 50
-        }
-     ]
+    "TXT": "v=spf1 mx:mail.gh0stp4wz.is-a.dev ip4:45.82.121.115 -all",
+    "MX": [
+      {
+        "target": "mail.gh0stp4wz.is-a.dev",
+        "priority": 10
+      }
```

the default priority is 10
```suggestion
    "MX": [
        "mail.gh0stp4wz.is-a.dev"
```